sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -95,6 +95,18 @@ object ScalaJSPluginInternal {
       "Files used to compute this value (can be used in FileFunctions later).",
       KeyRanks.Invisible)
 
+  val scalaJSJSDOMDirectory = SettingKey[File](
+    "scalaJSJSDOMDirectory",
+    "Directory in which jsdom should be installed.",
+    KeyRanks.Invisible
+  )
+
+  val scalaJSJSDOMVersion = SettingKey[String](
+    "scalaJSJSDOMVersion",
+    "Version of jsdom to install",
+    KeyRanks.Invisible
+  )
```

Comment:
Can't we hide these within the `jsEnv` setting (especially if they are internal keys) and just use the default values inside `JSDOMNodeJSEnv`?

(And maybe add arguments for them, but if you didn't want to expose these in the first place, no need).